examples/allennlp/allennlp_simple.py

```diff
@@ -93,34 +98,24 @@ def objective(trial):
     lr = trial.suggest_float("lr", 1e-3, 1e-1, log=True)
     optimizer = torch.optim.SGD(model.parameters(), lr=lr)
 
-    data_loader = torch.utils.data.DataLoader(
-        train_dataset, batch_size=16, collate_fn=allennlp.data.allennlp_collate
-    )
-    validation_data_loader = torch.utils.data.DataLoader(
-        valid_dataset, batch_size=64, collate_fn=allennlp.data.allennlp_collate
-    )
-
     serialization_dir = os.path.join(MODEL_DIR, "trial_{}".format(trial.number))
     trainer = allennlp.training.GradientDescentTrainer(
         model=model,
         optimizer=optimizer,
-        data_loader=data_loader,
+        data_loader=train_data_loader,
         validation_data_loader=validation_data_loader,
         validation_metric="+" + TARGET_METRIC,
         patience=None,  # `patience=None` since it could conflict with AllenNLPPruningCallback
         num_epochs=30,
         cuda_device=DEVICE,
         serialization_dir=serialization_dir,
-        epoch_callbacks=[AllenNLPPruningCallback(trial, "validation_" + TARGET_METRIC)],
+        callbacks=[AllenNLPPruningCallback(trial, "validation_" + TARGET_METRIC)],
     )
     metrics = trainer.train()
     return metrics["best_validation_" + TARGET_METRIC]
 
 
 if __name__ == "__main__":
-    if version.parse(allennlp.__version__) < version.parse("1.0.0"):
-        raise RuntimeError("AllenNLP>=1.0.0 is required for this example.")
-
```

Comment:
[question] why is this exception removed instead of updated?

Reply:
Oh, thank you for pointing out. I unexpectedly removed it.
d37728c restores and updates version constraint.